app/models/ta.rb

```diff
@@ -37,11 +37,14 @@ def percentage_grades_array(assignment)
     grades = []
 
     if assignment.assign_graders_to_criteria
-      criteria_data = self.criteria.where('criteria.assessment_id': assignment.id).sum(:max_mark)
-      return [] if criteria_data.zero?
+      criteria_ids = self.criterion_ta_associations.where(assessment_id: assignment.id).pluck(:criterion_id)
```

Comment:
Made a comment on the other PR as well about the same thing, but we can't change this logic because we use criteria_data later when finding mark data, so we still want to keep track of the ids. Rather we should probably just change the name to `criteria_ids.